aws/s3/s3.go

```diff
@@ -78,6 +78,28 @@ func getConfig() (aws.Config, error) {
 	if os.Getenv("AWS_REGION") == "" {
 		return aws.Config{}, errors.New("AWS_REGION is not set")
 	}
+
+	awsEndpoint := os.Getenv("CUSTOM_AWS_ENDPOINT_URL")
+	if awsEndpoint != "" {
+		customResolver := aws.EndpointResolverWithOptionsFunc(func(service, region string, options ...interface{}) (aws.Endpoint, error) {
+			return aws.Endpoint{
+				PartitionID:       "aws",
+				URL:               awsEndpoint,
+				SigningRegion:     "us-east-1",
```

Comment:
Would this matters? If this doesn't affect the functionality then I'd suggest we try use `os.Getenv("AWS_REGION")`

Reply:
Yeah so the reason why it's "us-east-1" is because that's the only value that I found that would work with SQS...any other region I tried, you get the following error:
```
operation error SQS: SendMessage, https response error StatusCode: 400, RequestID: PSNZLT26VKSVK8HTKIG2AM0AG18VQ2JJ9YFHGHVI3B6FI64VNRZ3, api error AWS.SimpleQueueService.NonExistentQueue: The specified queue does not exist for this wsdl version.
```

So I decided to hard-code us-east-1 in there...what we could do is still do `os.Getenv("AWS_REGION")`, but stipulate in the `kit` documentation/comments that you need to set this region to `us-east-1` in the env vars if you plan on using SQS. 

What do you think?